api/app/tests/hfi/test_hfi_endpoint_request_stored.py

```diff
@@ -35,7 +36,7 @@ def given_request_have_one_stored(monkeypatch,
 
     def mock_get_most_recent_updated_hfi_request(_, __):
         """ Returns mocked WFWXWeatherStation with fuel types. """
-        return HFIRequest(request=stored_request_json)
+        return HFIRequest(request=json.dumps(stored_request_json))
```

Comment:
shouldn't this be `json.loads()` instead of dumps()?

Reply:
Nope. The request property needs a string, so taking a dictionary, and turning it into a string.